apps/agreement/contracts/Agreement.sol

```diff
@@ -307,7 +308,7 @@ contract Agreement is IAgreement, AragonApp {
 
         emit ActionSubmitted(id, msg.sender);
 
-        // Pay court transaction fees
+        // Pay app transaction fees
```

Comment:
I'm wondering if we can come up with a better comment for this, as "app transaction fees" is a bit vague.

Maybe "action submission fees" is a better concept?